abs.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
+  - https://staging.continuum.io/prefect/fs/phonetics-feedstock/pr1/43564a7
```

Comment:
while this may work, this is not the best way of doing this, because if a new commit is pushed to that pr, this url will change. ideally you'd use a custom channel in anaconda.org

Reply:
yes, I am aware of that! In fact, I leave it there only as long as that PR does not change and periodically check for changes.